BUILD.md

```diff
@@ -10,13 +10,14 @@ git clone https://github.com/dm-vdo/kvdo
 
 Build the uds library:
 ~~~
-make -C $BUILD_DIR/vdo/utils/uds/
+cd   $BUILD_DIR/kvdo
+make -C /usr/src/kernels/`uname -r` M=`pwd`
 ~~~
 
 Get and build the LZ4 library:
 ~~~
 git clone https://github.com/lz4/lz4
```

Comment:
After the "cd" above, the current directory is $BUILD_DIR/kvdo so this will put lz4 there.